rosidl_generator_c/test/test_interfaces.c

```diff
@@ -24,53 +24,74 @@
 #include "rosidl_generator_c/primitives_array_functions.h"
 #include "rosidl_generator_c/string_functions.h"
 
-#include "rosidl_generator_c/msg/various__struct.h"
+
 #include "rosidl_generator_c/msg/bool__struct.h"
+#include "rosidl_generator_c/msg/bounded_array_nested__struct.h"
+// #include "rosidl_generator_c/msg/bounded_array_primitives__struct.h"
 #include "rosidl_generator_c/msg/byte__struct.h"
 #include "rosidl_generator_c/msg/char__struct.h"
+#include "rosidl_generator_c/msg/constants__struct.h"
```

Comment:
Out of curiousity, is there any reason we prefer to `#include "rosidl_generator_c/msg/constants__struct.h"` (and the corresponding functions.h) rather than including constants.h, which will pull them both in?